var/spack/repos/builtin/packages/c/test/hello.c

```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
+int main()
+{
+	printf ("Hello world from C!\n");
```

Comment:
Style:

```
    printf("Hello world from C!\n");
```

Reply:
I guess if we are getting technical about this, this exists: https://www.python.org/dev/peps/pep-0007/

Reply:
I don't think we should use a style guide on the code we're using to test the compilers. There is value in throwing unusual combinations of things at a compiler (although we obviously are not testing comprehensively enough to do so in a rigorous manner).

Reply:
These tests come from John G. I'm assuming that most of the weirdness in the test code is an accumulation of individual errors that some compiler has had over the course of his career.